src/ripple/app/ledger/tests/common_ledger.cpp

```diff
@@ -435,28 +435,61 @@ trust(TestAccount& from, TestAccount const& issuer,
 void
 close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL)
 {
+    // This version of close_and_advance uses wall time.
+    using namespace std::chrono;
+    auto const epoch_offset = days(10957);  // 2000-01-01
+    auto const closeTime = time_point_cast<seconds>  // now
+        (system_clock::now() - epoch_offset).
+        time_since_epoch().count();
+
+    // Pass the time delta to the other close_and_advance.
+    close_and_advance (
+        ledger, LCL, closeTime - ledger->getParentCloseTimeNC());
+}
+
+void
+close_and_advance(Ledger::pointer& ledger,
+    std::shared_ptr<Ledger const>& LCL, std::uint32_t advSeconds)
```

Comment:
We don't need a whole function just to pass an empty container of transactions, just make callers do it.